activesupport/lib/active_support/messages/metadata.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+require "time"
+
+module ActiveSupport
+  module Messages
```

Comment:
We should probably put a `#:nodoc:` here as well, so this module won't show up in docs.